docs/questions/sharing/exporting-results.md

```diff
@@ -43,6 +45,23 @@ You can export a dashboard as a PDF. Click on the three dot menu (**...**) and s
 
 The PDF will only include screenshots of the charts as they are visible on the dashboard.
 
+### Exporting results of a dashboard card
+
+To export the results of a particular card, hover over the dashboard card, click on the three dot menu (**...**), and select **Download results**.
+
+![Export results of a dashboard card](../images/download-card-results.png)
+
+From here you can select:
+
+- .csv
+- .json
+- .xlsx
+- .png (if a chart)
+
+To export the raw, unformatted results, hold down the `option` key for Macs, or `Alt` key for Windows, then click on the download file format.
```

Comment:
I couldn't do this part. I held the Option key and downloaded a CSV file for a question, but the results were the same. Does the UI change when we hold the Option key?
<img width="822" alt="Screenshot 2024-04-05 at 2 56 40 PM" src="https://github.com/metabase/metabase/assets/154609371/0f474a0a-d1b1-4159-9044-5246918770cb">

Reply:
The option to download raw, unformatted results is only available in 49.3 and up.